pkg/kademlia/kademlia.go

```diff
@@ -340,6 +340,7 @@ func recalcDepth(peers *pslice.PSlice) uint8 {
 // connect connects to a peer and gossips its address to our connected peers,
 // as well as sends the peers we are connected to to the newly connected peer
 func (k *Kad) connect(ctx context.Context, peer swarm.Address, ma ma.Multiaddr, po uint8) error {
+	k.logger.Infof("attempting to connect to peer %s", peer)
```

Comment:
there is already a logline relating to this in `kademlia.go#L170`

Reply:
Perfect. So you suggest to remove this log and use the log on line 170? I would, however, prefer to use my wording, as this contains less jargon and is therefore easier to understand for the average early-adopter user. Besides, I would use the info log level, as this is an important action that tells the user a story on what the node is doing.

Reply:
I had a look at it, and I would suggest to leave my log line and remove the debug log line on `170`. The reason for this is that `func (k *Kad) connect(ctx context.Context, peer swarm.Address, ma ma.Multiaddr, po uint8)` may also be called from another place and you want to have the info in the function itself to always have the correct info.

Reply:
Please change the logline in `L170` to display the information you log here. I don't mind changing the loglevel. Why do we need to log the same information twice?

> may also be called from another place

but it is not called from anywhere else.

either remove `L170` or change it please

Reply:
> either remove L170 or change it please

￼This was done here: https://github.com/ethersphere/bee/pull/1145/commits/865f64f733c32a8f268a537d770cffc664c0301d